### PR TITLE
Other/readme

### PR DESCRIPTION
**Pull request by roeb**
Add README markdowns to the repo base and the typescript connector

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
 
+## About 
+
+<a href="https://www.kalypso.cloud/" target="_blank">Kalypso</a> is a platform to design workflows for connecting and automating processes with a wide variety of applications and services, including those in the real estate industry. 
+
+Kalypso is based on connectors that allow anyone to become part of the platform. The basis of a connector is a <a href="https://docs.kalypso.cloud/docs/partners/connector-definition" target="_blank">Definition</a> that describes the triggers and actions of a connector.
+
+In the documentation, there is a <a href="https://docs.kalypso.cloud/docs/partners/tutorial/" target="_blank">tutorial</a> that explains how to create a connector step by step.
+
+In this repository there are implementations of the tutorial for connectors in the different languages.
```
Comment (thoemmi) [inline README.md:16]: in ~the~ different languages

File: connectors/typescript/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# TypeScript Kalypso Connector
+
+This connector is based on NodeJS, TypeScript and Express. It is an example of a connector created based on the <a href="https://docs.kalypso.cloud/docs/partners/tutorial/" target="_blank">tutorial</a> in the docs.
+
+## File structure
+
+The connector is splitted in seperate folders:
```
Comment (thoemmi) [inline connectors/typescript/README.md:7]: sep**a**rate

File: connectors/typescript/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+- `requests` contains the RequestHandlers for Actions and Triggers registrations
+- `store` contains the InMemory Subscription Store to save and remove registered subscriptions
+- `types` contains all required types
+
+## Running the connector
+
+To start the connector, run the following command:
```
Comment (thoemmi) [inline connectors/typescript/README.md:15]: I guess the API Key for the connector must be provided first and added to an .env file or something like that

File: README.md (hunk 1 above)
Comment (roeb) [reply README.md]: Fixed

File: connectors/typescript/README.md (hunk 2 above)
Comment (roeb) [reply connectors/typescript/README.md]: Fixed

File: connectors/typescript/README.md (hunk 3 above)
Comment (roeb) [reply connectors/typescript/README.md]: I have added the API Key information

**Review by thoemmi — APPROVED** [review]

**Pull request merged**